src/scripts/itemDiscuss/dimItemDiscuss.directive.js

```diff
@@ -0,0 +1,66 @@
+import angular from 'angular';
+
+angular.module('dimApp')
+  .directive('dimItemDiscuss', ItemDiscuss);
+
+function ItemDiscuss() {
+  return {
+    controller: ItemDiscussCtrl,
+    controllerAs: 'vm',
+    bindToController: true,
+    scope: {},
+    templateUrl: require('./dimItemDiscuss.directive.template.html')
```

Comment:
We've been using `import` instead of `require`. Also note that now we use `template` instead of `templateUrl` because we're inlining templates.